crmsh/corosync_config_format.py

```diff
@@ -231,17 +233,20 @@ def on_dict(self, node):
                     self.on_dict(value)
                     del self._path_stack[-1]
                     self.__write_indent(len(self._path_stack))
-                    self._ofile.write('}\n')
+                    self._ofile.write('}\n\n')
                 case list(_):
                     self._path_stack.append(key)
                     self.on_list(value)
                     del self._path_stack[-1]
                 case _:
-                    self.__write_indent(len(self._path_stack))
-                    self._ofile.write(key)
-                    self._ofile.write(': ')
-                    self.on_value(value)
-                    self._ofile.write('\n')
+                    if key.startswith(COMMENT_PREFIX):
+                        self._ofile.write(f'# {value}\n')
```

Comment:
```suggestion
                        self.__write_indent(len(self._path_stack))
                        self._ofile.write('# ')
                        self._ofile.write(value)
                        self._ofile.write('\n')
```

1. Writing to buffered io directly tends to provide better performance.
2. Indentation is needed